### ISSUE TITLE
Position in Mentoring Queue seems to be increasing

### ISSUE THREAD

**Issue by mobad** [exercism/exercism#4906]
I've been keeping an eye on my position in the mentoring queue for my solution to one of the later core exercises in the Rust track.
https://exercism.io/my/solutions/233418ec8cee48369f4e5e97711dffd8
I started at position ~220 and my position seems to be gradually fluctuating upwards.
I'm currently at 231 but it changes frequently.
Here's a short excerpt of the changes in queue position https://i.imgur.com/dpLKBna.png

I wonder if it could be displaying the size of the queue, not my position in it?

@iHiD

**Comment by ryanbarry** [comment]
i'm noticing the same on an [earlier exercise](https://exercism.io/my/solutions/4dae6b859f004f49b49ebf40e3f4cfcf) in the rust track (Bob) – currently sitting at around 85 (i think it was 81 when i submitted). could it be that the overall queue is ordered by exercise?

**Comment by iHiD** [comment]
@mobad @ryanbarry Could you tell me if things look better now please?

**Comment by mobad** [comment]
I'm at 252 now, so no.

**Comment by ryanbarry** [comment]
same here, i'm up to 91 now; but it says i'm still at the same spot in the exercise queue

**Comment by iHiD** [comment]
Should be fixed by https://github.com/exercism/website/pull/658

**Comment by iHiD** [comment]
Deployed. Does that look better for you both now?

**Comment by mobad** [comment]
Yup, I'm at 210 now. I'll keep an eye on it today and see how it changes, thanks!

**Comment by iHiD** [comment]
Thank you both for the helpful information, and sorry for the bug :)

**Comment by mobad** [comment]
Unfortunately it still looks like my place in the queue is increasing.
I was 210 earlier today and now I'm 213.
https://i.imgur.com/369nVNx.png

**Comment by iHiD** [comment]
There might be small fluctuations like this if a mentor times-out and so their issues go back into the queue. I'll leave this open for another day or two and we can see what happens.
